csss-site/src/events/frosh/templates/frosh/2022/frosh.html

```diff
@@ -48,6 +48,12 @@ <h1>Frosh Week 2022</h1>
     <p>Something sweet is coming!</p>
     <img src="{% static 'frosh_static/2022/images/frosh2022_logo_1000x1000.png' %}" alt="Frosh 2022 Logo" id="logo" class="center"/> 
 </section>
+<div class="center">
+    <video controls preload="metadata">
+        <!-- Need to use SFU Vault because of GitHub 100 MB file size limit. This link currently points to a file hosted in Oliver's (Liversticks) SFU account. -->
+        <source src="http://vault.sfu.ca/index.php/s/cRoboB5qowvz6uk/download" type="video/mp4"/>
```

Comment:
do whatever update you need to do to redirect to https://youtu.be/kCjToAoN0fA

Reply:
Do you have the relevant HTML snippet generated per this page (https://support.google.com/youtube/answer/171780?hl=en)? I'll try using the bare URL in the source but it may not work directly.

Reply:
@modernNeo using the youtube URL directly will not work. Please look into the embed HTML. (I don't have access to the YouTube channel so I can't do it myself)

Reply:
https://www.wikihow.com/Embed-a-YouTube-Video

Reply:
Thanks for the clarification. Updated with the video embedded.